Source/Android/app/src/main/res/values/strings.xml

```diff
@@ -244,4 +244,6 @@
     <string name="header_controllers">Controllers</string>
 
     <string name="write_permission_needed">You need to allow write access to external storage for the emulator to work</string>
+
+    <string name="load_settings">Load Settings</string>
```

Comment:
"Load" doesn't seem like the right verb form to use here. The message should say something like "Loading Settings..." or maybe just "Loading..."